test/Constants.sol

```diff
@@ -17,7 +16,8 @@ contract Constants {
   uint256 constant MAX_REASONABLE_TIME_PERIOD = 302_400; // 6 weeks assume a 12 sec block time
 
   // we have not yet deployed the Radworks Bravo Governor
-  address constant DEPLOYED_BRAVO_GOVERNOR = 0x1111111111111111111111111111111111111111;
+  // @dev: this is a placeholder address and will be replaced with the deployed address
+  address constant DEPLOYED_BRAVO_GOVERNOR = address(0x0);
```

Comment:
(arbitrary comment)

- When looking at the Scopelint spec of PoolTogether I noticed a couple test cases were missing in Radworks.

│   ├──  Old Governor Cannot Send E T H After Proposal Succeeds
│   ├──  Old Governor Sends Token After Proposal Is Defeated

Reply:
Done: [2d0828d](https://github.com/ScopeLift/radworks-governor-upgrade/pull/22/commits/2d0828d6b7bb9186f9288613f2755725b4581f3b)